src/_mixins/use-form-item.ts

```diff
@@ -29,14 +30,14 @@ interface UseFormItemOptions<T> {
   mergedSize?: (formItem: FormItemInjection | null) => T
 }
 
-type UseFormItemProps<T> =
-  | {
-    size?: T
-  }
-  | {}
+interface UseFormItemProps<T> {
+  size?: T
+  disabled?: boolean
+}
 
 export interface UseFormItem<T> {
   mergedSizeRef: ComputedRef<T>
+  mergedDisabledRef: ComputedRef<boolean | undefined>
```

Comment:
```suggestion
  mergedDisabledRef: ComputedRef<boolean>
```

应该去掉 undefined，从这里出来的已经是最终的值了，默认都应该是 false。

顺带有某些类型你可能需要改一下，都是和这个相关但是用不到的。